src/Sylius/Component/Promotion/Checker/Rule/ItemTotalRuleChecker.php

```diff
@@ -20,7 +20,7 @@
     '1.13',
     'The "%s" class is deprecated and will be removed in Sylius 2.0. Use "%s" instead.',
     ItemTotalRuleChecker::class,
-    \Sylius\Component\Core\Promotion\Checker\Rule\ItemTotalRuleChecker::class,
+    'Sylius\Component\Core\Promotion\Checker\Rule\ItemTotalRuleChecker',
```

Comment:
But why? If fqcn syntax doesn't work anymore for some reason, it should be possible to add `use CartQuantityRuleChecker as CoreCartQuantityRuleChecker`